client/partials/form.html

```diff
@@ -33,9 +33,24 @@
 			</fieldset>
 		{{/ifCond}}
 	{{else}}
-		<label for="form-{{this.name}}" class="{{required this.required}}">{{{this.label}}}</label>
+		<label for="form-{{this.name}}" class="{{required this.required}}">
+			{{{this.label}}}
+			{{#if this.minWordCount}}
+				<small>(At least {{this.minWordCount}} words)</small>
+			{{/if}}
+			{{#if this.maxWordCount}}
+				<small>(Max {{this.maxWordCount}} words)</small>
+			{{/if}}
+			{{#if this.minCharacterCount}}
+				<small>(At least {{this.minCharacterCount}} characters)</small>
+			{{/if}}
+			{{#if this.maxCharacterCount}}
+				<small>(Max {{this.maxCharacterCount}} characters)</small>
+			{{/if}}
+			<small class="current-count"></small>
+		</label>
 		{{#ifCond this.type "textarea"}}
-			<textarea name="{{this.name}}" id="form-{{this.name}}" placeholder="{{this.placeholder}}" {{required this.required}}>{{this.value}}</textarea>
+			<textarea name="{{this.name}}" id="form-{{this.name}}" placeholder="{{this.placeholder}}" {{required this.required}} minlength="{{this.minCharacterCount}}" maxlength="{{this.maxCharacterCount}}" data-max-word-count="{{this.maxWordCount}}" data-min-word-count="{{this.minWordCount}}">{{this.value}}</textarea>

```

Comment:
Does this still add the `minlength` and other attributes even if those aren't defined in the JSON file?

Reply:
Yeah it does
Is it worth adding a Handlebars partial to only add the attribute if it is set?